utils/vscode/package.json

```diff
@@ -14,16 +14,13 @@
     "Programming Languages",
     "Snippets"
   ],
+  "type": "commonjs",
```

Comment:
I'm having trouble finding documentation for this field. What does it mean?

Reply:
CommonJS:
 ```js
let http = require("http");
```

Module/ ESM:
```js
import http from "http";
```